tests/test_cmdline.py

```diff
@@ -17,6 +17,7 @@
 from importlib_metadata import files as package_files
 
 from fawltydeps.main import UNUSED_DEPS_OUTPUT_PREFIX, VERBOSE_PROMPT, Analysis, version
+from fawltydeps.settings import default_ignore_unused
```

Comment:
```suggestion
from fawltydeps.settings import DEFAULT_IGNORE_UNUSED
```

Reply:
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/b6afa98689e4720130fad434d0ce13e2d2b44a26.